src/main/java/SergheiRubailo/Task8/Book.java

```diff
@@ -0,0 +1,25 @@
+package SergheiRubailo.Task8;
+
+public class Book {
+
+    String title;
+    String author;
+    double price;
+    String isbn;
+
+    public Book(String title, String author, double price, String isbn) {
+        this.title = title;
+        this.author = author;
+        this.price = price;
+        this.isbn = isbn;
+    }
+
+    public void displayDetails() {
+        System.out.println("Title: " + title + ", Author: " + author + ", Price: " + price + ", ISBN: " + isbn);
+    }
+
+    public void applyDiscount(double percentage) {
+        price = price * (1 - percentage / 100);
```

Comment:
did you understand why this price is calculated in this way ?

Reply:
Yes. (if I'm not messed up with logic :) )
As a first step, we need to convert percents to parts of a whole price: (percentage / 100).
Next, we subtract this value from the whole part: (1 - percentage / 100).
Next, we multiply the price to remain after subtraction part: price * (1 - percentage / 100).
And get the price with applied discount.
The new price is saved to the price of a Book object instance.

price = price * ( (100 - percentage) / 100 )
It can be written the next way, but in my opinion it's harder to read.

Thank you!